Resources/Prototypes/Entities/Structures/Power/Generation/generators.yml

```diff
@@ -0,0 +1,262 @@
+# Base Generator
+
+- type: entity
+  abstract: true
+  id: BaseGenerator
+  description: A high efficiency thermoelectric generator.
+  name: generator
+  placement:
+    mode: SnapgridCenter
+  components:
+  #- type: AmbientSound
+  #  range: 5
+  #  sound:
+  #    path: /Audio/Ambience/Objects/engine_hum.ogg
```

Comment:
why commented out?

Reply:
At the time there was no way for making it only play audio when it was on. Waiting on generators PR to add actual functionality to these beyond being a supplier.